deploy/eks/cloudbeat-ds.yaml

```diff
@@ -0,0 +1,331 @@
+# Deploy a cloudbeat instance per node for cis policy
```

Comment:
Do we want to refine this file as part of this PR ?

Reply:
@ofiriro3 I think it's best to have one yaml.
I'd add the env vars added for eks to the existing cloudbeat yaml file.

Reply:
I find it much easier to have a single YAML for each cloud provider. 

If you will pay close attention you would see that there are some differences. 
The fetchers that are being spawned are different as for some of the yaml file currently commented out because AWS won't allow you to do certain operations (like cluster-role that is being managed by AWS)